src/components/breadcrumb/BreadcrumbSvgPathOnly.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+export function BreadcrumbSvgPathOnly({
```

Comment:
- [ ] Rename to ChevronIcon that contains only the path
- [ ] No properties
- [ ] The icon should resize based on the SVG size

Reply:
done ish, one property left (color) and agreed to keep it as a react component for now